package.json

```diff
@@ -75,7 +73,9 @@
     "sinon-chai": "3.7.0",
     "uglify-js": "3.17.4",
     "wait-on": "8.0.3",
-    "wiremock": "2.35.0"
+    "wiremock": "2.35.0",
+    "jest": "^29.7.0",
```

Comment:
Can we remove these carets ^. We want to be pinning versions for our dependency management